automation/features/step-definitions/staking.steps.js

```diff
@@ -1,45 +1,72 @@
 const { When, Then } = require("@wdio/cucumber-framework");
+const stakingPage = require("../pageobjects/staking.page");
 let currentAmountOfTokensInWallet = "";
+let validatorName = "";
+/* eslint-disable  no-unused-vars */
+let tokensAssociatedInVegaWallet = "";
```

Comment:
chore (blocking): Remove unused vatrs don't just disable the linting rules for them. all of thse in this file

Reply:
returns unused but we actually assign values to them so not sure why it still returns 'unused vars'